src/Card/Card.js

```diff
@@ -11,12 +12,11 @@ function Card({ id, states, fullName, weatherInfo, images }) {
         <img key={images.id} src={images.url} alt={images.altText} />
       </div>
       <h3>Weather Info</h3>
-      {/* <p>{weatherInfo}</p> */}
-      <button>See More!</button>
+      <p>{weatherInfo}</p>
```

Comment:
excellent, its good to see you using that prop type, we may need to add it into our prop types though, havent read that far ahead yet